direct/src/directnotify/Notifier.py

```diff
@@ -285,7 +290,7 @@ def debugStateCall(self, obj=None, fsmMemberName='fsm',
             self.__print(string)
         return 1 # to allow assert self.notify.debugStateCall(self)
 
-    def debugCall(self, debugString=''):
+    def debugCall(self, debugString: object = '') -> int:
```

Comment:
Just for my curiosity, how does annotating something with `object` differ from not annotating it?

Reply:
The difference is that annotating as an object means none of that object's attributes can be used in the function. A more significant difference happens when a constant/variable is annotated with an object, than when the argument function is.

The correct annotation in this case though would likely be Any, which is equivalent to not having the annotation.

Reply:
https://docs.python.org/3/library/typing.html#the-any-type

Reply:
@Wizzerinus is correct that leaving a parameter unannotated is equivalent to annotating it with `Any`. Annotating a parameter with `object`, on the other hand, means that literally any object can be passed to the function. I've done that on purpose here, as all that's done with the argument is passing it to `str`, which can accept any object and return a string.

Reply:
Ahh, cool.  Thanks for the explanation.  Not sure the annotation has added value, but setting it to `object` seems fine, at least it makes it explicit that we accept non-strings here.